translations/base-cz.yaml

```diff
@@ -186,11 +185,11 @@ dialogs:
 
     resetKeybindingsConfirmation:
         title: Reset klávesových zkratek
-        desc: Vaše klávesové zkratky budou resetovány. Prosím potvrďte.
+        desc: Opravdu chcete vrátit klávesové zkratky zpět do továrního nastavení?
```

Comment:
this means "reset to factory settings" @tobspr is that OK? (there's no factory technically, I'd go with "default settings" - `původní nastavení`)

Reply:
Both are fine for me :)

Reply:
I went with "factory settings" just makes it abundantly clear you're about to reset back to default, but I guess `původní` would work as well.